### PR TITLE
Added short guide for connecting kratos to AzureAD

### PR DESCRIPTION
**Pull request by ernax78**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: docs/versioned_docs/version-v0.3/guides/sign-in-with-github-google-facebook-linkedin.mdx [hunk 1]
```diff
@@ -99,6 +99,64 @@ selfservice:
...
+http(s)://<domain-of-ory-kratos>:<public-port>/self-service/browser/flows/strategies/oidc/callback/<provider-id>
+```
+
+```json title="contrib/quickstart/kratos/email-password/oidc.azuread.jsonnet"
+# claims contains all the data sent by the upstream.
+local claims = std.extVar('claims');
+{
+  identity: {
+    traits: {
+      email: claims.email
```
Comment (aeneasr) [inline docs/versioned_docs/version-v0.3/guides/sign-in-with-github-google-facebook-linkedin.mdx:129]: Does AzureAD include `email_verified`? If so, we should probably add a warning here as we did for Google/GitHub: https://github.com/ory/kratos/pull/433/files#diff-2a62fee703dc56d70e23792e010a5a01L67-L73

Comment (ernax78) [reply docs/versioned_docs/version-v0.3/guides/sign-in-with-github-google-facebook-linkedin.mdx:129]: There is an optional claim, verified_primary_email that can be queried. 
https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-optional-claims

Comment (ernax78) [reply docs/versioned_docs/version-v0.3/guides/sign-in-with-github-google-facebook-linkedin.mdx:135]: @aeneasr - I read up on the issue on the Microsoft documentation, it seems like the same issue with unverified e-mails applies for Microsoft accounts as for GitHub, however, if you are using a corporate AzureAD (one you have control over), this does not apply and the email is to be considered as "verified". I did try to extract the verified_primary_email token that should theoretically work for this, however I did not have success at the moment (and it is not needed for my use-case). I think the documentation I submitted should be good to go with the warning / links attached.

Comment (aeneasr) [reply docs/versioned_docs/version-v0.3/guides/sign-in-with-github-google-facebook-linkedin.mdx:135]: Ok, sounds good!

File: docs/versioned_docs/version-v0.3/guides/sign-in-with-github-google-facebook-linkedin.mdx [hunk 2]
```diff
@@ -126,12 +126,20 @@ local claims = std.extVar('claims');
...
+      //
+      // If connecting only to your organization (one tenant), claims.email is safe to use if you have not actively disabled e-mail   
+      //verification during signup.
       email: claims.email
     },
   },
 }
 ```
 
+See https://docs.microsoft.com/en-us/azure/active-directory-b2c/user-flow-disable-email-verification for warnings.
```
Comment (aeneasr) [inline docs/versioned_docs/version-v0.3/guides/sign-in-with-github-google-facebook-linkedin.mdx:141]:
```suggestion
Also see ["Disable email verification during customer sign-up in Azure Active Directory B2C"] for warnings.
```

**Pull request merged**